kaztron/cog/dice.py

```diff
@@ -93,9 +114,9 @@ def __init__(self, bot):
 
             if isinstance(root_exc, errors.UnauthorizedChannelError):
                 logger.error("Unauthorized use of command in #{1}: {0}"
-                    .format(cmd_string, ctx.message.channel.name))
+                             .format(cmd_string, ctx.message.channel.name))
                 await self.bot.send_message(ctx.message.channel,
-                    "Sorry, this command can only be used in {}".format(self.ch_dice.mention))
+                                            "Sorry, this command can only be used in {}".format(self.ch_dice.mention))
```

Comment:
why these whitespace revisions? This is different from the formatting convention used elsewhere in kaztron, and in this case it also makes the line exceed 100 characters - please don't! =P